lib/web_ui/lib/ui_web/src/ui_web/platform_view_registry.dart

```diff
@@ -19,6 +19,20 @@ final PlatformViewRegistry platformViewRegistry = PlatformViewRegistry();
 
 /// A registry for factories that create platform views.
 class PlatformViewRegistry {
+  /// The view type of the built-in factory that creates visible platform view
+  /// DOM elements.
+  ///
+  /// There's no need to register this view type with [PlatformViewRegistry]
+  /// because it is registered by default.
+  static const String defaultVisibleViewType = '_default_document_create_element_visible';
```

Comment:
❤️ and without the "k" prefix.

![tumblr_70976177f0cdc973a43fba42c9c04621_20bc424a_400](https://github.com/flutter/engine/assets/1255594/70b96c4c-3a1d-4029-aece-62ac7fbc025d)